charts/zabbix/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 name: zabbix
 version: 3.4.4
```

Comment:
Hi @sa-ChristianAnton !
Thank you for your contribution.
Sorry for being late.
I was on vacation. 
Could you change the chart version to 3.4.5?